packages/boutique/backend/package.json

```diff
@@ -8,7 +8,7 @@
   },
   "dependencies": {
     "@google-cloud/logging-winston": "^6.0.0",
-    "@interledger/open-payments": "^6.7.1",
+    "@interledger/open-payments": "^6.8.0",
```

Comment:
Is there an issue with the `6.8.2` version ?

Reply:
probably not, this is the version in rafiki

Reply:
will go for ^ 6.8.2 then, rafiki already uses 6.8.2 just that ^6.8.0 is in the package json